api/lib/schema.js

```diff
@@ -327,6 +327,7 @@ const typeDefs = gql`
   type ClaimRewardsTx {
     amounts: [Coin]!
     from: [String]!
+    rewards: [Reward] # Polkadot only
```

Comment:
```suggestion
    rewards: [Reward] # Substrate only
```

Reply:
Same comment: I would reduce the name "Substrate"

Reply:
ok, fair enough